imap_data_access/file_validation.py

```diff
@@ -500,6 +501,7 @@ def is_valid_cr(input_cr: str) -> bool:
     "metakernel": "mk",
     "thruster": "activities",
     "lagrange_point": "spk",
+    "high_precision_planetary_constants": "bcp",
```

Comment:
It looks like the standard directory for planetary constants files is "pck".
```suggestion
    "earth_attitude": "pck",
```